docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
```

Comment:
This file doesn't seem to be used within the build.  What is it used for?

Reply:
docker-compose.yml is the file used for deploying the CIMI server in containers. In can be seen as documentation

Reply:
If it is documentation, then it is probably better in the ssdocs documentation (either the contributor's guide or administrator's guide). If there's a reason to keep it in the source code, then it should be moved to the cimi (cimi-resources, cimi-rpm, ?) submodule.

Reply:
maybe. But this is just for testing since the SS deployment doesn't rely on the Dockerized version of CIMI.

maybe put this into the README?

Reply:
If this is intended purely for developers, then it can be added to a README for the CIMI server.